toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -384,6 +384,13 @@ func PopulateInstallRoot(installChroot *safechroot.Chroot, packagesToInstall []s
 
 	installRoot := filepath.Join(rootMountPoint, installChroot.RootDir())
 
+	if config.IsKickStartBoot && config.IsIsoInstall {
+		err = configuration.UpdatePackageRepo(installChroot, config)
+		if err != nil {
+			return
+		}
+	}
```

Comment:
Do we want to error out (or at least print a warning) if we define custom repos but don't select kickstart and iso flags? Since this is something that can only be validated at runtime (since we don't set the ISO flag ahead of time) it might be confusing to debug. I'd lean towards an error so there is no way to miss the warning.

Reply:
Makes sense to me

Reply:
Added a check in imageconfigvalidator

Reply:
Resolved